src/pages/Wallets.js

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button';
+
+import '../App.css'
+import WalletForm from '../components/WalletForm';
+import { Modal } from 'react-bootstrap';
 
 function Wallets() {
+  const [showModal, setShowModal] = useState(false);
+
+  const onCreateWalletClick = () => setShowModal(true);
+  const handleCloseModal = () => setShowModal(false)
+
   return (
     <div className='wallets'>
-      Wallets
+      <Button
+        variant="primary"
+        size="lg"
+        onClick={onCreateWalletClick}
+      >
+        Create Wallet
+      </Button>
+
+      <Modal show={showModal} onHide={handleCloseModal} >
```

Comment:
Puede ir directamente 
```suggestion
      <Modal show={showModal} onHide={ () => setShowModal(false) } >
```